django_setup_configuration/constants.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+
+basic_field_descriptions = {
+    models.BooleanField: "True, False",
+    models.CharField: "string",
+    models.EmailField: "string representing an Email address (foo@bar.com)",
+    models.FileField: (
+        "string representing the (absolute) path to a file, "
+        "including file extension: {example}".format(
+            example="/absolute/path/to/file.xml"
+        )
```

Comment:
I don't see why we use `.format` instead of just putting it in one string, but I guess it doesn't really make a difference